lib/webauthn/attestation_statement/base.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require "cose"
```

Comment:
Would it work if we limit this to `"cose/algorithm"`?

Reply:
we also have an explicit use of `COSE::Error` and although it works without requiring `cose/error` (because one of the algorithm files requires it), we should require it here as well I think

I'll replace this require in favor of `require 'cose/algorithm'` + `require 'cose/error'`